padertorch/contrib/examples/source_localization/distance_estimator/download.py

```diff
@@ -0,0 +1,97 @@
+"""
+Download the room impulse response (RIR) database, a JSON file containing voice
+activity information (VAD) for LibriSpeech as well as LibriSpeech itself,
+if it is required.
+
+Both database storage paths can either be a path to the data files or a path
+for where to download the files, depending on the needs and prerequisites.
+In the former case, it is checked whether the existing data files meet the
+requirements or have to be downloaded.
+
+If only one path is specified, only the corresponding database is downloaded.
+
+The JSON for the VAD information regarding the LibriSpeech train_clean_100
+dataset has to be downloaded in any case, so its path must be specified.
+The path for this file is assumed to be the same as for the LibriSpeech
+database, unless otherwise stated, which can be done by adding the suffix
+"with vad_json_path=/PATH/TO/VAD_JSON" to the call.
+
+May be called with:
+"python -m padertorch.contrib.examples.source_localization.distance_estimator.
+download with rir_path=/PATH/TO/RIR_DB libri_path=/PATH/TO/LIBRISPEECH"
+
+"""
+
+from pathlib import Path
+
+from sacred import Experiment
+
+from paderbox.io.download import download_file, download_file_list
+
+
+ex = Experiment()
+
+
+@ex.config
+def config():
+    libri_path = None
+    rir_path = None
+    vad_json_path = None
+    if vad_json_path is None:
+        vad_json_path = Path(libri_path) / "speech_activity_librispeech.json"
+        msg = "Please specify a path where to store the VAD information JSON"
+        assert vad_json_path is not None, msg
+    libri_url = 'http://www.openslr.org/resources/12/train-clean-100.tar.gz'
```

Comment:
Please download dev-clean (and use it for the validation) and test-clean (and use it for the evaluation), too. These data set parts have also to be taken into account in  create_jsons.py